### PR TITLE
RAD-93: Update wcs attribute information

### LINKED ISSUE #181: Add/Update wcs attribute information
_Issue [RAD-93](https://jira.stsci.edu/browse/RAD-93) was created on JIRA by [David Davis](https://jira.stsci.edu/secure/ViewProfile.jspa?name=ddavis):_

Add/Update wcs attribute information with the origin most are likely from "Science Data Formatting (SDF)"

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [RAD-93](https://jira.stsci.edu/browse/RAD-93)

Closes #181 

This PR updates wcs keywords and attribute information

## Tasks
- [ ] Update or add relevant `rad` tests.
- [ ] Update relevant docstrings and / or `docs/` page.
- [x] Does this PR change any schema files?
  - [x] Schema changes were discussed at RAD Review Board meeting.
- [x] Does this PR change any API used downstream? (If not, label with `no-changelog-entry-needed`.)
  - [ ] Write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types).
  - [ ] Start a `romancal` regression test (https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/rad@<branch>"`).
  - [x] Update relevant `roman_datamodels` utilities and tests.

<details><summary>News fragment change types:</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

File: src/rad/resources/schemas/wcsinfo-1.0.0.yaml [hunk 1]
```diff
@@ -33,12 +85,16 @@ properties:
       datatype: float
       destination: [WFIExposure.v3_ref, GuideWindow.v3_ref]
   vparity:
-    title: Relative Rotation Between Ideal and Telescope Axes
+    title: Relative Rotation Between the Ideal and Telescope Ideal Axes
```
Comment (PaulHuwe) [inline src/rad/resources/schemas/wcsinfo-1.0.0.yaml:88]:
```suggestion
    title: Relative Rotation Between the Ideal and Telescope Axes
```

File: src/rad/resources/schemas/wcsinfo-1.0.0.yaml [hunk 2]
```diff
@@ -3,14 +3,60 @@
...
+           'WFI05_FULL', 'WFI06_FULL', 'WFI07_FULL', 'WFI08_FULL',
+           'WFI09_FULL', 'WFI10_FULL', 'WFI11_FULL', 'WFI12_FULL',
+           'WFI13_FULL', 'WFI14_FULL', 'WFI15_FULL', 'WFI16_FULL',
+           'WFI17_FULL', 'WFI18_FULL']
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
+    archive_catalog:
+      datatype: nvarchar(12)
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/wcsinfo-1.0.0.yaml:29]: How certain are we that this is a complete list? We've only allowed for 2 extra characters. If we think this list might expand in the future we might want to allow for extra characters now.

Comment (schlafly) [reply src/rad/resources/schemas/wcsinfo-1.0.0.yaml:29]: We should include at least WFI_CEN, BORESIGHT, and CGI_CEN.  There's one other placeholder in the SIAF called WFI_TITLE that I think we can safely ignore.

I think with those additions the list would be complete, and I don't think it's very likely to change so we could stick with the enum?  I am also partial to just doing strings with 20 characters to give us more flexibility.

**Review by jbrookens — APPROVED** [review]

File: src/rad/resources/schemas/wcsinfo-1.0.0.yaml (hunk 2 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/wcsinfo-1.0.0.yaml:29]: Keywords added and string length expanded.

**Pull request merged**